index.html

```diff
@@ -19,5 +20,39 @@ <h2>Share your favorite recipes!</h2>
         <h2>Browse the recipes of others!</h2>
         <p>Let yourself be inspired by the favorite recipes of cooking enthusiasts from all over the world 🌎. You can even tell them what you think about it! </p>
     </div>
+    <p>If you want to check out popular recipe Sites have a look at these ones:</p>
+    <ul>
+        <li><a href="https://www.chefkoch.de/">Chefkoch</a></li>
+        <li><a href="https://www.edeka.de/rezepte/suche.jsp">Edeka</a></li>
+        <li><a href="https://www.rewe.de/rezepte/">REWE</a></li>
+    </ul>
+
+    <div>
+        <table>
+                <tr>
+                    <td>First</td>
+                    <td>Second</td>
+                    <td>Third</td>
```

Comment:
```suggestion
                    <td>Name</td>
                    <td>Number of Ingredients</td>
```
Könntest du das und dann die dazugehörigen Tabelen-Daten anpassen?

Das würde dann zu der submit-recipe.html Seite passen.

Reply:
Done